AppCenterData/AppCenterData/MSData.h

```diff
@@ -66,9 +66,11 @@ typedef void (^MSPaginatedDocumentsCompletionHandler)(MSPaginatedDocuments *docu
  * @param completionHandler Callback to accept downloaded document.
  */
 + (void)readDocumentWithID:(NSString *)documentID
-                      type:(Class)documentType
+              documentType:(Class)documentType
                  partition:(NSString *)partition
-         completionHandler:(MSDocumentWrapperCompletionHandler)completionHandler;
+         completionHandler:(MSDocumentWrapperCompletionHandler)completionHandler // clang-format off
```

Comment:
Unfortunately the formatter would break the `NS_SWIFT_NAME` if line too long. No choice but to deactivate it.